### PR TITLE
Minor tests script updates

### PR DESCRIPTION
**Pull request by caub**
minor changes on tests

### CODE REVIEW

File: test/web-platform-tests/start-wpt-server.js [hunk 1]
```diff
@@ -69,6 +69,6 @@ function pollForServer(url) {
     })
     .catch(err => {
       console.log(`WPT server at ${url} is not up yet (${err.message}); trying again`);
-      return q.delay(500).then(() => pollForServer(url));
+      return new Promise(r => setTimeout(r, 500)).then(() => pollForServer(url));
```
Comment (TimothyGu) [inline test/web-platform-tests/start-wpt-server.js]: You could have a `promisify(setTimeout)` as well.

**Comment by TimothyGu** [comment]
Seems like this broke Node.js v6.x due to a lack of `util.promisify`...

**Comment by caub** [comment]
oh :(, let's temporarily close and reopen in 1 year when we can skip node v6 maybe

**Comment by domenic** [comment]
I find most of these changes less clear than the original. Eliminating the q dependency would be nice if we could, but we can't, so yeah, let's close this.

**Pull request closed without merging**